data/companies.json

```diff
@@ -544,6 +544,23 @@
         "youtube": null
       }
     },
+    {
+      "name": "Karobar App",
+      "logoName": "karobar_app.png",
```

Comment:
The filename format is `karobarapp.png`

Reply:
Just made the changes. You can review it.